package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "s3-streamlogger",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
please don't bump the version in PRs – I'll bump the version just before tagging the release

Reply:
No problem. I'll send a new commit with these changes soon.